spikeinterface/sorters/launcher.py

```diff
@@ -46,6 +44,111 @@ def _run_one(arg_list):
 _implemented_engine = ('loop', 'joblib', 'dask')
 
 
+def run_sorter_by_property(sorter_name,
+                           recording,
+                           grouping_property,
+                           working_folder,
+                           mode_if_folder_exists='raise',
+                           engine='loop',
+                           engine_kwargs={},
+                           verbose=False,
+                           docker_image=None,
+                           **sorter_params):
+    """
```

Comment:
Could you use *args and **kwargs here to avoid to maintain the list at sevarl place.
For instance pIerre is adding a new kwarg in separate PR and it will not be included.

Reply:
The signature is slightly different. It's a mix between the `run_sorter` and the `run_sorters`. The API should be similar to the `run_sorter` (one), but since internally it uses the `run_sorters` it also has some args from it (but not all! for example, `with_output` is set to True, etc)